src/components/RedditFooter/RedditFooter.tsx

```diff
@@ -1,84 +1,71 @@
-import React, { Component } from 'react';
-import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import React, { Component } from "react";
```

Comment:
All these should be ' instead of ". Maybe you could add eslint on a different PR.